src/components/RootApp/RootApp.tsx

```diff
@@ -124,6 +124,22 @@ const RootApp = memo((props: RootAppProps) => {
           </Suspense>
           <ChromeProvider bundle={bundleTitle}>
             <QuickStartContainer {...quickStartProps}>
+              const drawerProps: QuickStartContainerProps = {
+                markdown: {
+                    extensions: [
+                      // Variable substitution example
+                      // This replaces the strings [PROJECT_NAME]
+                      {
+                        type: 'output',
+                        filter: function(html: string) {
+                          html = html.replace(/\[PRODUCT_NAME\]/g, 'App Studio');
```

Comment:
html.replace(/\[(.+)]{{(product_name) ([\w-]+)}}/g, ...) over here.  Let's see what we need to get the product_name from the backend in your other PR.  https://github.com/gtrivedi88/quickstarts/pull/1